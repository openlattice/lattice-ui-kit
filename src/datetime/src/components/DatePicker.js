import { DatePicker } from '@atlaskit/datetime-picker';
import withProps from '../../../components/withProps';
import emotionStyles from '../../../components/emotionStyles';

export const props = {
  selectProps: {
    styles: emotionStyles
  }
};

const LatticeDatePicker = withProps(DatePicker, props);

export default LatticeDatePicker;
