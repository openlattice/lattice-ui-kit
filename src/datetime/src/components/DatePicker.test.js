import React from 'react';
import { mount, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
<<<<<<< HEAD
import { DatePicker } from '@atlaskit/datetime-picker';
// import DatePicker from './DatePicker';
=======
import DatePicker, { props } from './DatePicker';
>>>>>>> feature/emotion-date-time

describe('DatePicker', () => {

  it('render matches snapshot', () => {
    const tree = shallow(<DatePicker />);
    expect(toJson(tree)).toMatchSnapshot();
  });

  it('should render internal DatePicker', () => {
    const wrapper = mount(<DatePicker />);
    expect(wrapper.find('DatePicker').exists()).toBeTruthy();
  });

  it('internal DatePicker should have selectProps', () => {
    const wrapper = mount(<DatePicker />);
    const datePickerProps = wrapper.find('DatePicker').props();
    expect(datePickerProps.selectProps).toEqual(props.selectProps);
  });

});
