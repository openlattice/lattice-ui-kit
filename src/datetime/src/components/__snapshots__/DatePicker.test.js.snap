// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker render matches snapshot 1`] = `
<<<<<<< HEAD
<AnalyticsContext
  data={
    Object {
      "componentName": "datePicker",
      "packageName": "@atlaskit/datetime-picker",
      "packageVersion": "6.3.9",
=======
<ForwardRef
  selectProps={
    Object {
      "styles": Object {
        "clearIndicator": [Function],
        "container": [Function],
        "control": [Function],
        "dropdownIndicator": [Function],
        "indicatorSeparator": [Function],
        "indicatorsContainer": [Function],
        "menu": [Function],
        "menuPortal": [Function],
        "option": [Function],
        "singleValue": [Function],
      },
>>>>>>> feature/emotion-date-time
    }
  }
>
  <ForwardRef />
</AnalyticsContext>
`;
